pom.xml

```diff
@@ -123,27 +123,152 @@
         <!-- narayana spring boot version -->
         <narayana-spring-boot.version>3.0.0.redhat-00017</narayana-spring-boot.version>
 
-
         <!-- versions -->
+        <aalto-xml-version>1.3.2</aalto-xml-version>
         <aether-version>1.0.2.v20150114</aether-version>
         <arquillian-container-se-managed-version>1.0.2.Final</arquillian-container-se-managed-version>
         <arquillian-version>1.7.0.Alpha10</arquillian-version>
-        <avro-version>1.11.0</avro-version>
-        <groovy-version>4.0.13</groovy-version>
+        <avro-version>1.12.0</avro-version>
+        <groovy-version>4.0.22</groovy-version>
         <graal-sdk-version>22.3.2</graal-sdk-version>
         <jakarta-jaxb-version>4.0.0</jakarta-jaxb-version>
+        <jakarta-el-version>6.0.1</jakarta-el-version>
+        <jakarta-resource-version>2.0.0</jakarta-resource-version>
         <jaxb-version>2.3.0</jaxb-version>
+        <jna-version>5.13.0</jna-version>
+        <jobjc-version>2.8</jobjc-version>
+        <kryo-version>2.24.0</kryo-version>
         <maven-compiler-plugin-version>3.11.0</maven-compiler-plugin-version>
         <maven.compiler.release>${jdk.version}</maven.compiler.release>
         <maven-javadoc-plugin-version>3.10.0</maven-javadoc-plugin-version>
         <maven-surefire-plugin-version>3.1.2</maven-surefire-plugin-version>
         <mycila-license-version>3.0</mycila-license-version>
+        <org-json-json-version>20231013</org-json-json-version>
+        <parsson-version>1.0.5</parsson-version>
+        <perfmark-version>0.27.0</perfmark-version>
         <springdoc-version>2.2.0</springdoc-version>
         <surefire.version>${maven-surefire-plugin-version}</surefire.version>
-        <swagger-parser-v3-version>2.1.10</swagger-parser-v3-version>
-	    <cyclonedx-maven-plugin-version>2.8.1</cyclonedx-maven-plugin-version>
+        <swagger-parser-v3-version>2.1.20</swagger-parser-v3-version>
+        <cyclonedx-maven-plugin-version>2.8.1</cyclonedx-maven-plugin-version>
+
+        <tomcat-version>10.1.28</tomcat-version>
+        <snappy-java-version>1.1.10.5</snappy-java-version>
+        <woodstox-stax2-api-version>4.2.1</woodstox-stax2-api-version>
+
+        <!-- Added for productized build - plexus versions used by patch maven plugin -->
+        <openshift-maven-plugin-version>1.16.2.redhat-00002</openshift-maven-plugin-version>
```

Comment:
since Jkube 1.17.0 is GA do we want to use start productization with the updated minor version?

Reply:
Updated